### DISCUSSION TITLE
Problem with flagship model on Windows

### DISCUSSION THREAD

**Discussion by xXCISSOUXx** [mathoudebine/turing-smart-screen-python#75] [Original Windows app (USBMonitor / ExtendScreen)]
Hello,

I did receive the turing version which works perfectly on my laptop with usbmonitor.exe, and there, I have just received the flagship version for my desktop PC, except that with ExtendScreen.exe in version 3.0beta, it does not work, it not read data from pc, it turns on, it is connected, but no data. (no yellow triangle in usb in device manager)

If I take the flagship with ExtendScreen.exe on my laptop, it works.

If I take the turing version on my desktop pc with usbmonitor.exe it works.

How to do, apart from swapping machines, knowing that I find ExtendScreen.exe poorer in themes and settings.

I tried to download the v2.0 of xuanfang (like we see on pictures), under original vendor apps section, but it sends me back to the v3.0beta version.

An idea ?

Thanks in advance.

**Comment by mathoudebine** [comment]
Hello @xXCISSOUXx here is the v2.5 of ExtenScreen that i saved from the official link before the 3.0 beta was uploaded. You can try it and see if it works for you:
[XFZX.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/9932032/XFZX.zip)

If not, you need to contact the seller of your screen

(-> Moving this issue to a discussion as it is not an issue with this project)

**Reply by xXCISSOUXx** [reply]
Hello, i will try your link, thanks.
The seller, how to explain ... he sell, but he can't explain anything, he  would refund me, i refused because it works on another laptop.
No support, no official site.
Thanks for your answer.

**Comment by xXCISSOUXx** [comment]
Hi, sorry, i've posted in the wrong section .

Then i've tried the 2.5 version, and same issue, it's connected, but screen don't show PC's info.

I don't understand why it work on my (older) laptop, and not on my desktop computer ....
